.travis.yml

```diff
@@ -14,7 +14,6 @@ before_script:
   - psql -c 'create database move_mil_test;' -U postgres
   - cp config/database.yml{.sample,}
   - cp config/secrets.yml{.sample,}
-  - bundle exec rails db:setup
```

Comment:
This was causing issues on CI since it loads the seed data instead of the data from the test fixtures.